prow/npj/npj.go

```diff
@@ -115,3 +116,27 @@ func BatchSpec(p config.Presubmit, refs kube.Refs) kube.ProwJobSpec {
 	}
 	return pjs
 }
+
+// EnvironmentForSpec returns a mapping of environment variables
```

Comment:
I'm not a huge fan of shortening stuff unless there's a need -- if you feel strongly I'll chop off the `ironment` :)